### ISSUE TITLE
Button does not announce "disabled"

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30934]
## Description
When using a screen reader disabled buttons do not announce that they are disabled.

This issue is covered in-depth by parent issue #30840, please check there for more details.

## React Native version:
v0.63

**Comment by amarlette** [comment]
Thank you @huzaifaaak for helping improve React Native's accessibility! Closing this issue since the PR #31001 resolves it.
